### DISCUSSION TITLE
Falla Laragon despues de actualizar apache, Mysql, y php

### DISCUSSION THREAD

**Discussion by carlosromero683** [leokhoa/laragon#800] [General]
![image](https://github.com/leokhoa/laragon/assets/52896962/3f076a09-4d08-47cf-a4df-ac36aaca4239)

**Comment by luisAntonioLAGS** [comment]
Descarga y copia estos paquetes a las carpetas correspondientes, recuerda que cuando es mysql en la version 8 es más complicado hacer un update, por lo que te recomendaría eliminar la carpeta mysql-8 dentro de data antes de inicializar laragon después de que hayas actualizado los paquetes. 
Aquí te dejo un enlace en donde comprimí de las versiones más recientes de los paquetes que tengo, en el comprimido incluí apache, php y mysql (inclui la version 8.0.37 ltsc ya que la 8.4 no esta soportada en laragon).
https://usersdrive.com/5s2sz5zq192b.html
